packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -960,7 +1019,7 @@ void main() {
     // Default text field (without leading icon).
     await tester.pumpWidget(buildTest(themeData, menuChildren, label: const Text('label')));
 
-    final Finder label = find.text('label');
+    final Finder label = find.text('label').first;
```

Comment:
Why do we add `.first` here? Did we add another label?

Reply:
Yes this PR duplicates the label for width computation.
It adds it to `DropdownMenuBody` children list, similarly to the menu items, the text field and the leading/trailing buttons which are already duplicated.
It might require updating existing tests (in Google testing for instance).

Reply:
Ah! That makes sense. Thanks for the explanation! I hope Google testing won't break, but let's see=)